modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -786,7 +786,7 @@
                           "trackedItemId": 325525,
                           "trackedItemStatus": "NEEDED",
                           "uploaded": false,
-                          "uploadsAllowed": false
+                          "uploadsAllowed": true
```

Comment:
this function is working now due to a recent change merged Monday. this updates the docs to work correctly.